cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -683,3 +715,81 @@ def test_custom_button_unassigned_behavior_objs(
             view = navigate_to(obj, "Details")
             btn = Button(view, button.text)
             assert btn.is_displayed if dest is ViaSSUI else not btn.is_displayed
+
+
+@pytest.mark.customer_scenario
+@pytest.mark.meta(automates=[1628727])
+@pytest.mark.parametrize("context", [ViaUI, ViaSSUI])
+def test_custom_button_expression_ansible_service(
+    appliance, context, vis_enb_button_service, order_ansible_service_in_ops_ui
+):
+    """ Test custom button on ansible service as per expression enablement/visibility.
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/4h
+        caseimportance: medium
+        casecomponent: CustomButton
+        startsin: 5.9
+        testSteps:
+            1. Create custom button group on Service object type
+            2. Create a custom button with expression (Tag)
+                a. Enablement Expression
+                b. Visibility Expression
+            3. Navigate to object Detail page
+            4. Check: button should not enable/visible without tag
+            5. Check: button should enable/visible with tag
+
+    Bugzilla:
+        1628727
+        1509959
+        1513498
+        1755229
+    """
```

Comment:
I can see this BZ(https://bugzilla.redhat.com/show_bug.cgi?id=1755229) is in Assigned state and test coverage is +. Can we add this BZ to coverage marker?

Reply:
@ganeshhubale we have separate test case for  `BZ-1755229`. It was creating problem to create test setup in `module scope` so just i added `TODO` comment and also include that bug in Bugzilla tag so i know this is affecting this test.